pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.ruff]
+# allow longer line length
+line-length = 95
+
+# don't be as strict with formatting in migrations
+extend-exclude = ["ppa/*/migrations/*.py"]
```

Comment:
interesting

Reply:
it was complaining about long line length (but not fixing it automatically), and I couldn't figure out how to skip just that rule!